File: Solve/A1/A1-030/A1-030.py
n = int(input())
nums = list(map(int, input().split()))

result = [] 
for i in range(n):
    a = nums[2*i]
    b = nums[2*i+1]
    result.append(max(a, b))

if(n==1): print(max(a,b))
else:
    expression = ' + '.join(map(str, result))
    total = sum(result)
    print(f"{expression} = {total}")

File: Solve/A1/A1-030/A1-030.c
#include <stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int sum=0;
    for(int i=0;i<n;i++){
        int x,y;
        scanf("%d %d",&x, &y);
        int mx;
        if (x > y) mx = x;
        else if (x < y) mx = y;
        else mx = x;
        sum+=mx;
        if (i!=n && n!=1) printf("%d",mx);
        if(i!=n-1) printf(" + ");
        else if(n!=1)  printf(" = ");
    }
    printf("%d",sum);
}
